util/io.go

```diff
@@ -17,13 +17,37 @@
 package util
 
 import (
+	"errors"
 	"fmt"
 	"io"
 	"os"
 	"path/filepath"
 	"strings"
+	"syscall"
 )
 
+func MoveFile(src, dst string) error {
+	err := os.Rename(src, dst)
+	var errno syscall.Errno
+	if err != nil && errors.As(err, &errno) && errno == syscall.EXDEV {
+		// os.Rename() failed because src and dst are on different filesystems
+		// Must manually copy and delete the original file
+		return manuallyMoveFile(src, dst)
+	}
+	return err
+}
+
+func manuallyMoveFile(src, dst string) error {
```

Comment:
```suggestion
func moveFile(src, dst string) error {
```

Reply:
Renamed to `moveFileByCopying` in d21751413